src/data/services/EnterpriseDataApiService.js

```diff
@@ -123,6 +123,47 @@ class EnterpriseDataApiService {
     });
   }
 
+  static fetchEnrolledLearnersForInactiveCourses(options) {
+    const { enterpriseId } = store.getState().portalConfiguration;
+    const queryParams = {
+      page: 1,
+      page_size: 50,
+      has_enrollments: true,
+      active_courses: false,
+      all_enrollments_passed: true,
+      extra_fields: ['enrollment_count', 'course_completion_count'],
+      ...options,
+    };
+
+    const analyticsUrl = `${EnterpriseDataApiService.enterpriseBaseUrl}${enterpriseId}/users/?${qs.stringify(queryParams)}`;
+    const jwtToken = getAccessToken();
+
+    return httpClient.get(analyticsUrl, {
+      headers: {
+        Authorization: `JWT ${jwtToken}`,
+      },
+    });
+  }
+
+  static fetchEnrolledLearnersForInactiveCoursesCsv() {
+    const { enterpriseId } = store.getState().portalConfiguration;
+    const jwtToken = getAccessToken();
+    const queryParams = {
+      has_enrollments: true,
+      active_courses: false,
+      all_enrollments_passed: true,
+      extra_fields: ['enrollment_count', 'course_completion_count'],
```

Comment:
We would also need to convert `extra_fields` to array in  `fetchEnrolledLearnersCsv` method.

Reply:
Done